kotlintest-assertions/src/main/kotlin/io/kotlintest/throwablehandling/AnyThrowableHandling.kt

```diff
@@ -0,0 +1,49 @@
+package io.kotlintest.throwablehandling
+
+import io.kotlintest.Failures
+
+/**
+ * Verifies that a block of code throws any [Throwable]
+ *
+ * Use function to wrap a block of code that you want to verify that throws any kind of [Throwable], where using
+ * [shouldThrowAny] can't be used for any reason, such as an assignment of a variable.
+ *
+ * If you want to verify a specific [Throwable], use [shouldThrowExactlyUnit].
+ *
+ * If you want to verify a [Throwable] and any subclass, use [shouldThrowUnit].
+ *
+ * @see [shouldThrowAny]
+ */
+inline fun shouldThrowAnyUnit(block: () -> Unit) = shouldThrowAny(block)
```

Comment:
Docs are great, but I think the double spacing is hard to read.

Reply:
What do you mean with double spacing? The space between lines?

Reply:
Never mind, it was just my screen.